### PR TITLE
fix schedule database

### PR DESCRIPTION
**Pull request by namframgia**

### CODE REVIEW

**Comment by namframgia** [comment]
https://edu-redmine.framgia.vn/issues/2852

File: app/Models/Schedule.php [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Schedule extends Model
+{
+    protected $fillable= [
+        'day_month_id', 'user_id', 'status'
```
Comment (BunnyPi04) [inline app/Models/Schedule.php]: https://viblo.asia/p/cac-convention-thuong-gap-khi-moi-bat-dau-training-php-tai-framgia-bJzKmgq6l9N#_27-mang-va-chuoi-12

File: app/Repositories/BatchRepository.php [hunk 2]
```diff
@@ -57,4 +101,33 @@ public function update(array $data, $id)
...
+
+    public function destroy($id)
+    {
+        DB::beginTransaction();
+        try {
+            $batch = Batch::findOrFail($id);
+            $users = $batch->users;
+            foreach ($users as $user) {
+                $user->dayMonths()->detach();
+                User::destroy($user->id);
```
Comment (BunnyPi04) [inline app/Repositories/BatchRepository.php]: ko nên gọi model trong vòng lặp nhé
=> https://laracasts.com/discuss/channels/eloquent/how-to-delete-multiple-records-using-laravel-eloquent

File: app/Models/Schedule.php [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Schedule extends Model
+{
+    protected $fillable= [
+        'day_month_id',
+        'user_id',
+        'status'
```
Comment (BunnyPi04) [inline app/Models/Schedule.php:12]: https://viblo.asia/p/cac-convention-thuong-gap-khi-moi-bat-dau-training-php-tai-framgia-bJzKmgq6l9N#_27-mang-va-chuoi-12

**Pull request merged**